examples/server/README.md

```diff
@@ -48,14 +48,12 @@ To get started right away, run the following command, making sure to use the cor
 ### Unix-based systems (Linux, macOS, etc.):
 
 ```bash
-./server -m models/7B/ggml-model.bin -c 2048
+./server -m models/7B/ggml-model.gguf -c 2048
 ```
 
 ### Windows:
 
 ```powershell
-server.exe -m models\7B\ggml-model.bin -c 2048
-```
 
```

Comment:
Looks like this was accidentally removed instead of only changing the model extension.

````suggestion
server.exe -m models\7B\ggml-model.gguf -c 2048
```
````